test-data/unit/semanal-namedtuple.test

```diff
@@ -138,10 +138,6 @@ MypyFile:1(
 from collections import namedtuple
 N = namedtuple('N') # E: Too few arguments for namedtuple()
 
-[case testNamedTupleWithTooManyArguments]
```

Comment:
This test stopped working because this error is now caught during type checking rather than semanal.